tools/tidyall

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env perl
```

Comment:
If we add a new tool, I would like to have all the options that the old tool provides, e.g.
```
Usage:
 tidy [-c|--check] [-f|--force] [-o|--only-changed] [-l|--list] [path/to/file]

Options:
 -h, -?, --help       display this help
 -c, --check          Only check for style check differences
 -f, --force          Force check even if tidy version mismatches
 -o --only-changed    Only tidy files with uncommitted changes in git. This can
                      speed up execution a lot.
-l --list             List files tidy would touch
 path/to/file         When passing a file as argument, tidy will run perltidy
                      wether it is added to the git tree or not
```
And if this is done in perl, I suggest to use Getopt::Long for parsing command line options, it's a core module.

Reply:
It's not a new tool _per se_, but rather removing the bash wrapper to expose the real `tidyall` invocation.

I'm not very inclined on argument translation (basically translate `--only-changed` to `--git` and `--check for --check-only`), but if friction is a reasonable big factor, I guess we can do the translation when preparing `ARGV` for `tidyall`.

Reply:
I just saw that tidyall already has a shortcut for `--git`: `-g` and for `--list`: `-l`. So that leaves check-only, and I think we can live with that long option :)